package.json

```diff
@@ -11,6 +11,7 @@
     "@nivo/core": "^0.79.0",
     "@nivo/line": "^0.79.1",
     "@nivo/radar": "^0.79.1",
+    "@octokit/core": "^4.1.0",
```

Comment:
can you explain why adding this library is necessary?

Reply:
we use octokit to fetch from github api, should we use fetch api instead?

Reply:
yes.  we should stick to the same method the rest of the repo uses.  no need to add complexity to the app.